IMAGE:
  WIDTH: 640
  HEIGHT: 480

XYCAR:
  Y_OFFSET: 390
  Y_GAP: 25
  Y_GAIN: 1.35
  START_SPEED: 5.1
  MAX_SPEED: 5.1
  MIN_SPEED: 5.1
  SPEED_CONTROL_THRESHOLD: 35.0
  ACCELERATION_STEP: 0.2
  DECELERATION_STEP: 0.05

PID:
  P_GAIN: 0.47
  I_GAIN: 0.000
  D_GAIN: 0.007

CANNY:
  LOW_THRESHOLD : 100
  HIGH_THRESHOLD : 150

HOUGH:
  MIN_PIXEL: 40
  MIN_LINE: 35
  MAX_GAP: 10

TOPIC:
  QUEUE_SIZE: 1
  SUB_IMG_NAME: /usb_cam/image_raw/
  SUB_SCAN_NAME: /scan
  SUB_YOLO_NAME: /yolov3_trt_ros/detections
  PUB_MOTOR_NAME: /xycar_motor

LIDAR:
  FRONT_ANGLE: 45.0
  FRONT_DEPTH: 0.3
  SIDE_DEPTH: 0.25
  FRONT_OBS_THRESH: 50 # 63
  SIDE_OBS_THRESH: 10

OBJECT:
  LABELS: [Left, Right, Crosswalk, Stop, Car, Red, Green, Yellow]
  YOLO_RESOLUTION: 416
  XY_DEPTH: 115.0

DEBUG: True